bedrock/firefox/templates/firefox/new/desktop/thanks.html

```diff
@@ -14,6 +14,8 @@
 
 {% block extrahead %}
   {{ super() }}
+  {{ css_bundle('protocol-call-out') }}
+  {{ css_bundle('protocol-emphasis-box') }}
```

Comment:
**Note:** I added this CSS bundle to the wrong `thanks.html` file in this PR, however in this PR, I correct it as both of the `thanks.html` files require the `emphasis-box` import anyway.